style.css

```diff
@@ -1 +1,291 @@
-/*  Add your styling here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  color: #FFFFFF;
+}
+
+body{
+  background: #dde5b6;
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-template-rows: 150px 20px auto 200px;
+  grid-template-areas: 
+    "hd"
+    "nav"
+    "sect"
+    "foot";
+}
+
+header {
+  background: #84a59d;
+  grid-area: hd;
+}
+
+header span {
+  font-family: cursive;
+  font-weight: bolder;
+  color: #dde5b6;
+  font-size: 1.rem;
+  position: absolute;
+  top: 4.5rem;
+  right: 2rem;
+  max-width: 11rem;
+}
+
+nav{
+  display: none;
+}
+
+section{
+  grid-area: sect;
+  margin: 0 1rem;
+  gap: 0.8rem;
+  display: grid;
+  grid-template-row: repeat(3, uto);
+}
+
+.hero h1{
+color: #FFFFFF;
+font-weight: bolder;
+}
+
+.hero p{
+  color: #FFFFFF;
+  font-weight: bolder;
+  margin: 1rem 0;
+}
+
+.cake img{
+  object-fit: cover;
+  width: 100%;  
+  height: 19rem;
+}
+
+.photos{
+  display:grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 0.9rem;
+}
+
+.photos img{
+  object-fit: cover;
+  height: 10rem;
+  width: 100%;
+}
+
+
+
+footer{
```

Comment:
Hi Wanda, 
Well and clear code structure, following some rules I suggest not leaving these spaces. Create a good habit from the beginning

Reply:
Thanks!